src/SprykerEco/Yves/Computop/Handler/PostPlace/AbstractPostPlacePaymentHandler.php

```diff
@@ -38,6 +39,10 @@ public function handle(QuoteTransfer $quoteTransfer, array $responseArray)
     {
         /** @var \Generated\Shared\Transfer\ComputopSofortInitResponseTransfer $responseTransfer */
         $responseTransfer = $this->converter->getResponseTransfer($responseArray);
+        if ($quoteTransfer->getPayment() === null) {
```

Comment:
From what I've learned in code, this IF is not required.
It will be always TRUE.

Please check other places as well.

Reply:
That will be allwasy FALSE, not required anymore.